### ISSUE TITLE
Apps taking too long to publish

### ISSUE THREAD

**Issue by mrbid** [flathub/flathub#5132]
What exactly is the update schedule? The claimed 4 hours would seem a major under-estimate currently lol.

It's not a big deal I was just wondering if anyone could clarify more exact timing or explain why it's takes so long currently.

**Comment by bbhtt** [comment]
Seems normal now.

**Comment by bbhtt** [comment]
Can you post which apps these are?

**Comment by bbhtt** [comment]
> https://flathub.org/apps/com.tuxpusher.TuxPusher
> https://flathub.org/apps/com.tuxpusher.TuxPusherAF
> https://flathub.org/apps/xyz.woxel.Woxel
> https://flathub.org/apps/com.voxdsp.VoxelPaintPro
> https://flathub.org/apps/com.voxdsp.PoryDrive

Failed due to server being down more or less around the same time. I've started rebuilds for all

Rebuild: 

https://buildbot.flathub.org/#/builders/6/builds/112496
https://buildbot.flathub.org/#/builders/6/builds/112497
https://buildbot.flathub.org/#/builders/6/builds/112498
https://buildbot.flathub.org/#/buildrequests/449927
https://buildbot.flathub.org/#/buildrequests/449929

**Comment by bbhtt** [comment]
Also, fyi, the server ran out of space and data had to be moved from faster SSDs to slower drives. So things will take a while to get published.

**Comment by bbhtt** [comment]
All were published, I'll keep the issue open until the situation with the server is resolved.

**Comment by jakobjakobson13** [comment]
Could someone also check org.freedesktop.Sdk.Extension.texlive/23.08, publish/54612 ? The manifest was updated the day before yesterday and I think something in the publishing went wrong.

**Comment by bbhtt** [comment]
> Could someone also check org.freedesktop.Sdk.Extension.texlive/23.08, publish/54612 ? The manifest was updated the day before yesterday and I think something in the publishing went wrong.

Seems to be published and on the latest commit:

```
flatpak remote-info flathub org.freedesktop.Sdk.Extension.texlive//23.08

TeX Live Sdk extension - LaTeX packages and tools

        ID: org.freedesktop.Sdk.Extension.texlive
       Ref: runtime/org.freedesktop.Sdk.Extension.texlive/x86_64/23.08
      Arch: x86_64
    Branch: 23.08
   Version: 20210325
Collection: org.flathub.Stable
  Download: 5.8 GB
 Installed: 9.1 GB

    Commit: 81a1a3471a500c72b1e870a893e6e2dfefb420f7eeb83d878795679ceb075a04
    Parent: dee23d6c0cd2fbcb63c28dadddecf7f4b99203c83d8408fb6058ad087cff5b9b
   Subject: Fix perllibs manifest and readme file (#154) (088135f3)
      Date: 2024-04-04 09:42:40 +0000
```

**Comment by bbhtt** [comment]
> Also, fyi, the server ran out of space and data had to be moved from faster SSDs to slower drives. So things will take a while to get published.

Temporarily solved by moving part of the data back to SSDs.
